pycryptobot.py

```diff
@@ -168,7 +168,7 @@ def getAction(now: datetime = datetime.today().strftime('%Y-%m-%d %H:%M:%S'), ap
         action = 'WAIT'
 
     # if disabled, do not buy within 3% of the dataframe close high
-    if app.disableBuyNearHigh() is True and (price > (df['close'].max() * 0.97)):
+    if last_action != 'BUY' and app.disableBuyNearHigh() is True and (price > (df['close'].max() * 0.97)):
```

Comment:
Should this not be last_action == 'SELL'? It's the same thing but the state after SELL is BUY and that is what we want.